test_containers/compose.yaml

```
@@ -0,0 +1,31 @@
+      test: curl --fail http://localhost:8069/web/login || exit 1
+      start_period: 10s
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  db:
+    image: ghcr.io/jabez007/odoo-client-lib
+    environment:
+      - POSTGRES_DB=postgres
+      - POSTGRES_USER=odoo
+      - POSTGRES_PASSWORD=odoo 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)